src/main/web/templates/html/t9-1.html

```diff
@@ -305,7 +305,7 @@ <h3 class="flush float-left">Refine results</h3>
 						</div>
 						<div class="box__content">
 							<label class="field flush field--spaced field--text">
-								<span class="panel panel--separated--bottom block">Contains</span>
```

Comment:
I don't think this file is used @jondewijones @rav-pradhan

Reply:
I wasn't sure so I changed it to be safe. Should I revert this?

Reply:
I think so. Ask Jon or Ravi to confirm but I don't think these are used. Maybe we can delete them?

Reply:
Hey, so from my understanding the html files aren’t used and it’s only handlebars files we need to update when making any template changes; it should be fine, then, to revert the changes in those. With regards to deletion, I’m not sure if there’s a reason that they’re still in the codebase (Jon perhaps best placed to answer that)